internal/backends/gl/event_loop.rs

```diff
@@ -32,6 +32,8 @@ pub trait WinitWindow: PlatformWindow {
         &self,
         constraints: (corelib::layout::LayoutInfo, corelib::layout::LayoutInfo),
     );
+    fn size(&self) -> winit::dpi::LogicalSize<f32>;
+    fn set_size(&self, size: winit::dpi::LogicalSize<f32>);
```

Comment:
I know existing function don't have documentation comment, but it would be better to have some.

In particular, it should say that this is a cached size and not the actual size.